sway-lib-std/src/inputs.sw

```diff
@@ -127,6 +127,7 @@ pub fn input_count() -> u16 {
     match tx_type() {
         Transaction::Script => __gtf::<u16>(0, GTF_SCRIPT_INPUTS_COUNT),
         Transaction::Create => __gtf::<u16>(0, GTF_CREATE_INPUTS_COUNT),
+        _ => revert(0),
```

Comment:
If you be nice to leave TODO here and in other places, since we plan to add new getters for corresponding fields=)

Reply:
This was temporary so I could compile the std-lib and run tests. These don't exist yet?